zipkin-server/src/main/java/zipkin/server/ZipkinQueryApiV1.java

```diff
@@ -43,6 +44,7 @@
  */
 @RestController
 @RequestMapping("/api/v1")
+@CrossOrigin("${zipkin.server.allowed-origins:''}")
```

Comment:
Provided we document cross origin in the zipkin-server/README I'm cool with
defaulting to *

Ex. a section similar to Self-Tracing

On Tue, Aug 16, 2016 at 9:58 PM, Tommy Ludwig <notifications@github.com>
wrote:

> In zipkin-server/src/main/java/zipkin/server/ZipkinQueryApiV1.java
> <https://github.com/openzipkin/zipkin/pull/1234#discussion_r74939322>:
>
> > @@ -43,6 +44,7 @@
> >   */
> >  @RestController
> >  @RequestMapping("/api/v1")
> > +@CrossOrigin("${zipkin.server.allowed-origins:''}")
>
> Making the default value here also * would be a good idea for those
> making custom Zipkin servers with @EnableZipkinServer or those using
> their own YAML file of properties that might not contain the new property.
> That is, if we want the default value to be * in these cases also. Do we?
> Or do we want people in those situations to make a conscious decision about
> the allowed origins for the query API?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openzipkin/zipkin/pull/1234/files/607ccc3512ff70c8d6dd946bca7903e0730f96d3#r74939322>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAD618AuBfDqSbM-nHOKkOWf-bQVRfqpks5qgcH6gaJpZM4JhLqg>
> .
>